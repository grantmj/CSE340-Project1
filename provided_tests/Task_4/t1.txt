TASKS 
    1 4

POLY 
    F(a) = 5a - 3a;

EXECUTE 
    INPUT x; 
    
INPUTS 
    1