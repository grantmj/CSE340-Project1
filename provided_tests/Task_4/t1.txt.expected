POLY - COMBINED MONOMIAL LISTS
	F(a) = 2 a;
